test/cmdlineargs.jl

```diff
@@ -597,6 +597,34 @@ let exename = `$(Base.julia_cmd()) --startup-file=no --color=no`
             "Int(Base.JLOptions().fast_math)"`)) == JL_OPTIONS_FAST_MATH_DEFAULT
     end
 
+    # --pkgimages with multiple cpu targets
+    @testset let v = readchomperrors(`$exename
+        --cpu-target='generic;sandybridge,-xsaveopt,clone_all;haswell,-rdrnd,base(1)'
+        --pkgimages=no`)
+
+        @test !v[1]
+        @test isempty(v[2])
+        @test v[3] == "ERROR: More than one command line CPU targets specified without a `--output-` flag specified"
+    end
+
+    @test success(`$exename
+        --cpu-target='generic;sandybridge,-xsaveopt,clone_all;haswell,-rdrnd,base(1)'
```

Comment:
Okay this should be propagated to the target process. https://github.com/JuliaLang/julia/blob/a23b29cb1a6013277e381092147fc527a44d924a/base/util.jl#L171

As should `JULIA_CPU_TARGET`. https://github.com/JuliaLang/julia/blob/a23b29cb1a6013277e381092147fc527a44d924a/base/loading.jl#L2071